src/main/resources/db/migration/v0.37.0.60__expression_atlas-load.sql

```diff
@@ -0,0 +1,87 @@
+
+insert into bulkload (id, backendbulkloadtype, name, internal, obsolete, group_id, dbdatecreated, bulkloadstatus)
+select nextval('bulkload_seq'), 'EXPRESSION_ATLAS','HUMAN Expression Atlas', false, false, id, now(), 'STOPPED'
+from bulkloadgroup where name = 'Expression Atlas Load';
+
+INSERT INTO bulkscheduledload (id, cronschedule, scheduleactive)
+SELECT id, '0 0 22 ? * SUN-THU', true FROM bulkload WHERE name = 'ZFIN Expression Atlas';
+
+INSERT INTO bulkscheduledload (id, cronschedule, scheduleactive)
+SELECT id, '0 0 22 ? * SUN-THU', true FROM bulkload WHERE name = 'SGD Expression Atlas';
+
+INSERT INTO bulkscheduledload (id, cronschedule, scheduleactive)
+SELECT id, '0 0 22 ? * SUN-THU', true FROM bulkload WHERE name = 'MGI Expression Atlas';
+
+INSERT INTO bulkscheduledload (id, cronschedule, scheduleactive)
+SELECT id, '0 0 22 ? * SUN-THU', true FROM bulkload WHERE name = 'WB Expression Atlas';
+
+INSERT INTO bulkscheduledload (id, cronschedule, scheduleactive)
+SELECT id, '0 0 22 ? * SUN-THU', true FROM bulkload WHERE name = 'FB Expression Atlas';
+
+INSERT INTO bulkscheduledload (id, cronschedule, scheduleactive)
+SELECT id, '0 0 22 ? * SUN-THU', true FROM bulkload WHERE name = 'RGD Expression Atlas';
+
+INSERT INTO bulkscheduledload (id, cronschedule, scheduleactive)
+SELECT id, '0 0 22 ? * SUN-THU', true FROM bulkload WHERE name = 'HUMAN Expression Atlas';
+
+insert into bulkurlload (id, bulkloadurl)
+select id, 'https://www.ebi.ac.uk/gxa/species/Danio_rerio/sitemap.xml?allEntries=true'
+from bulkload where name = 'ZFIN Expression Atlas';
+
+insert into bulkurlload (id, bulkloadurl)
+select id, 'https://www.ebi.ac.uk/gxa/species/Saccharomyces_cerevisiae/sitemap.xml?allEntries=true'
+from bulkload where name = 'SGD Expression Atlas';
+
+insert into bulkurlload (id, bulkloadurl)
+select id, 'https://www.ebi.ac.uk/gxa/species/Caenorhabditis_elegans/sitemap.xml?allEntries=true'
+from bulkload where name = 'WB Expression Atlas';
+
+insert into bulkurlload (id, bulkloadurl)
+select id, 'https://www.ebi.ac.uk/gxa/species/Mus_musculus/sitemap.xml?allEntries=true'
+from bulkload where name = 'MGI Expression Atlas';
+
+insert into bulkurlload (id, bulkloadurl)
+select id, 'https://www.ebi.ac.uk/gxa/species/Drosophila_melanogaster/sitemap.xml?allEntries=true'
+from bulkload where name = 'FB Expression Atlas';
+
+insert into bulkurlload (id, bulkloadurl)
+select id, 'https://www.ebi.ac.uk/gxa/species/Rattus_norvegicus/sitemap.xml?allEntries=true'
+from bulkload where name = 'RGD Expression Atlas';
+
+insert into bulkurlload (id, bulkloadurl)
+select id, 'https://www.ebi.ac.uk/gxa/species/Homo_sapiens/sitemap.xml?allEntries=true'
+from bulkload where name = 'HUMAN Expression Atlas';
+
+
+insert into resourcedescriptorpage (id, name, urltemplate, pagedescription, resourcedescriptor_id)
+select nextval('resourcedescriptorpage_seq'), 'expression_atlas', 'http://www.ebi.ac.uk/gxa/genes/[%s]', 'Expression Atlas', id
```

Comment:
This should be added to the resource descriptor file and not manually to the database. The load that runs will delete this.